MonoGame.Framework/Media/Album.cs

```diff
@@ -167,7 +168,13 @@ public UIImage GetAlbumArt(int width = 0, int height = 0)
 #elif ANDROID
         public Bitmap GetAlbumArt(int width = 0, int height = 0)
         {
-            var albumArt = MediaStore.Images.Media.GetBitmap(MediaLibrary.Context.ContentResolver, this.thumbnail);
+            Bitmap albumArt;
+            if (!OperatingSystem.IsAndroidVersionAtLeast (29)) {
```

Comment:
I tried writing a wrapper method for `IsAndroidVersionAtLeast` which takes the `BuildVersionCodes`  enum. But doing that stopped the compiler from ignoring the obsolete warnings which is kind of the point of this whole exercise. So I decided to stick with the `OperatingSystem.IsAndroidVersionAtLeast` api.